pom.xml

```diff
@@ -169,17 +169,17 @@
 
     <dependency>
       <groupId>io.prometheus</groupId>
-      <artifactId>simpleclient</artifactId>
+      <artifactId>prometheus-metrics-model</artifactId>
```

Comment:
Should we just import the BOM https://central.sonatype.com/artifact/io.prometheus/prometheus-metrics-bom

Reply:
We try to avoid that. It makes it hard to override dependencies etc.